test_tar_changes.py

```diff
@@ -75,3 +75,19 @@ def test_upload_source(delete_sources, mocked_run, mocked_sources):
     # flag was detected by argparse
     tarchanges.upload_source(osdist, 'test.tar', False)
     mocked_run.assert_called_once() # run() was not called a 2nd time
+
+@patch.object(git, 'get_commit_hashes', return_value = ["9e20ef1b14ac70dea53123"])
```

Comment:
remove the spaces around the equals sign here

Reply:
I have made the changes